src/hotspot/share/gc/g1/g1CardTable.hpp

```diff
@@ -103,8 +104,8 @@ class G1CardTable : public CardTable {
   // be inaccurate as it does not perform the dirtying atomically.
   inline size_t mark_region_dirty(size_t start_card_index, size_t num_cards);
 
-  // Mark the given range of cards as Scanned. All of these cards must be Dirty.
-  inline void mark_as_scanned(size_t start_card_index, size_t num_cards);
+  // Mark the given range of cards to "which". All of these cards must be Dirty.
```

Comment:
Maybe change the comment to "Mark the given range of dirty cards as..." or the function name to "mark_dirty_to"

Reply:
I think I fixed that.